src/views/RegisterForm/registerFormConstants.json

```diff
@@ -1,30 +1,30 @@
 [
     {
-        "placeholder": "Enter your first name",
+        "placeholder": "👤 Enter your first name",
```

Comment:
Please change to noun form: "First name"

idem for the other placeholders in RegisterForm